examples/messages/configured_remote.output.json

```diff
@@ -0,0 +1,28 @@
+{
+  "workflow_config": {
+    "Example": {
+      "inlinestr": "prefix{meta.foo}suffix",
+      "array": "{[$.meta.foo]}",
+      "object": "{{$.meta}}"
+    }
+  },
+  "cumulus_meta": {
+    "task": "Example",
+    "message_source": "local",
+    "id": "id-1234",
+    "message_bucket": "pq-test"
+  },
+  "meta": {
+    "foo": "bar",
+    "someKey": "someData"
+  },
+  "payload": {
+    "input": { "anykey": "anyvalue" },
+    "config": {
```

Comment:
I understand everything except for what is writing to `payload.config`

Reply:
So - the CMA takes an input configuration like: 

```
      CumulusConfig:
        buckets: '{$.meta.buckets}'
        provider: '{$.meta.provider}'
        collection: '{$.meta.collection}'
        stack: '{$.meta.stack}'
        downloadBucket: '{$.cumulus_meta.system_bucket}'
        duplicateHandling: '{$.meta.collection.duplicateHandling}'
        pdr: '{$.meta.pdr}'
        cumulus_message:
          outputs:
            - source: '{$.granules}'
              destination: '{$.meta.input_granules}'
            - source: '{$}'
              destination: '{$.payload}'
            - source: '{$.process}'
              destination: '{$.cumulus_meta.process}'
```

The `cumulus_message` key dictates what the input/outputs should be from the tasks, but the rest of it is all passed to the function - that's why in core all of our CMA tasks are referencing "event.config".

Reply:
The above all happens in loadNestedEvent in message_adapter.py around L341

Reply:
Yeah.... that's not even new behavior of the message adapter. Makes perfect sense 👍